src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const preparedQuery = query.toLowerCase().trim();
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    const preparedTitle = title.toLowerCase();
+    const preparedDescription = description.toLowerCase();
+
+    const isIncludeTitle = preparedTitle.includes(preparedQuery);
```

Comment:
it is better to move logic of lowerCasing and `includes` check to separate function

Reply:
Roger that!